webui-src/app/people/people_ownids.js

```diff
@@ -1,213 +1,379 @@
-let m = require('mithril');
-let rs = require('rswebui');
-let widget = require('widgets');
-let people_util = require('people/people_util')
+let m = require("mithril");
+let rs = require("rswebui");
+let widget = require("widgets");
+let people_util = require("people/people_util");
+
+function checksudo(data) {
+  if (data === "0000000000000000") return true;
+  return false;
+}
 
 const CreateIdentity = () => {
   // TODO: set user avatar
-  let name = '',
-    pgpPassword = '',
+  let name = "",
+    id = "",
+    pgpPassword = "",
     pseudonimous = false;
   return {
-    view: () => [
-      m('i.fas.fa-user-plus'),
-      m('h3', 'Create new Identity'),
-      m('hr'),
-      m('input[type=text][placeholder=Name]', {
+    view: (v) => [
+      m("i.fas.fa-user-plus"),
+      m("h3", "Create new Identity"),
+      m("hr"),
+      m("input[type=text][placeholder=Name]", {
         value: name,
-        oninput: e => (name = e.target.value),
+        oninput: (e) => (name = e.target.value),
       }),
-      'Type:',
       m(
-        'select',
+        "div",
         {
-          value: pseudonimous,
-          oninput: e => (type = e.target.value),
+          style: "display:inline; margin-left:5px;",
         },
         [
-          m('option[value=false]', 'Linked to your Profile'),
-          m('option[value=true]', 'Pseudonymous'),
-        ],
+          "Type:",
+          m(
+            "select",
+            {
+              value: pseudonimous,
+              style: "border:1px solid black",
+              oninput: (e) => {
+                if (e.target.value == "false") pseudonimous = false;
+                else pseudonimous = true;
```

Comment:
Always use strict equals for same data types.

Can also be made ternary:
```suggestion
                pseudonimous = e.target.value === "true" ? true: false;
```

Ideally we should be using an object to store and reference states, but it's only 2 options here, so it's fine.

Reply:
Okk @rottencandy, I will change it.